lib/verbs/conjugator.rb

```diff
@@ -193,7 +193,7 @@ def present_participle(infinitive)
       when /(^be|ye|oe|nge|ee)$/
         infinitive.to_s
       when /ie$/
-        "#{infinitive.to_s[0..-2]}y"
```

Comment:
@rossmeissl negative index here was confusing (for me?) so replaced it with `gsub`.

```ruby
irb(main):004:0> 'die'[0..-2]
=> "di"
```

Confusing when you have a Python background.
```python
>>> 'die'[0:-2]
'd'
```